org.nongnu.enigma.yml

```diff
@@ -40,12 +122,19 @@ modules:
         url: https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/2.49.0/graphviz-2.49.0.tar.gz
         sha256: 17ade6189beade7c0784090f283c1f78f6b286ec38e3d2e927d3e42d625d0e8d
 
-  - name: Xerxes-C
+  - name: Xerces-C
     buildsystem: cmake-ninja
     sources:
-      - type: archive
-        url: https://dlcdn.apache.org//xerces/c/3/sources/xerces-c-3.2.3.tar.gz
-        sha256: fb96fc49b1fb892d1e64e53a6ada8accf6f0e6d30ce0937956ec68d39bd72c7e
+      - type: git
+        url: https://github.com/apache/xerces-c
+        tag: v3.2.3
+        commit: cf1912ac95d4147be08aef4e78f894a3919277d9
+
+  - name: libpng
+    sources:
+      - type: git
+        url: https://github.com/glennrp/libpng
+        tag: v1.6.35
```

Comment:
does it really need to build libpng? it's in the runtime

Reply:
Probably then it could be removed, i will try tomorrow to build it locally without this and see if it works